### ISSUE TITLE
关于训练数据问题

### ISSUE THREAD

**Issue by aijianiula0601** [OlaWod/FreeVC#21]
你好，你提供的demo很好。我有个问题：提供出来的pretrain_models是只采用vctk数据训练而已吗？ 有加入其他训练数据训练吗？ 
我按照你的代码，用你代码的原始处理数据方式训练了vctk后进行训练，在v100机器，用两张卡，训练了一周，效果没有你公开的pretrain_models好。所以有疑惑。

1.训练的数据集只用vctk吗？
2.训练需要多长时间，几张卡呢？

**Comment by OlaWod** [comment]
Yes vctk only. To be specific, the pretrained models are trained with those in 'filelists/train.txt' using 'data_utils_old.py' on a single NVIDIA 3090 for 900k steps.

**Comment by OlaWod** [comment]
Hi, sorry to have wasted your training time. 😥 The code I uploaded had some problem and I've fixed it. (Hope there are no other problems)
